README.adoc

```diff
@@ -47,7 +47,7 @@ using, and developing Minishift:
 
 - https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started with Minishift]
 - https://docs.openshift.org/latest/minishift/using/index.html[Using Minishift]
-- https://docs.openshift.org/latest/minishift/developing/index.html[Developing Minishift]
+- https://docs.openshift.org/latest/minishift/contributing/developing.html[Developing Minishift]
```

Comment:
@LalatenduMohanty Then Text also changed to "Contributing to Minishift" instead "Developing Minishift" in readme. I just updated the link because it was broken.